ckanext/switzerland/plugins.py

```diff
@@ -255,33 +284,10 @@ def after_upload(self, context, resource_dict, dataset_dict):
 class OgdchOrganisationSearchPlugin(plugins.SingletonPlugin):
```

Comment:
do we still need these plugins if we are using the controllers now?

Reply:
i see that you removed them from setup.py and the config, so i guess we can remove the classes too. :)